components/event-publisher-proxy/pkg/sender/beb/beb_test.go

```diff
@@ -0,0 +1,221 @@
+package beb
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/env"
+	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/oauth"
+	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/sender"
+	testing2 "github.com/kyma-project/kyma/components/event-publisher-proxy/testing"
+)
+
+const (
+	// mock server endpoints.
+	eventsEndpoint        = "/events"
+	eventsHTTP400Endpoint = "/events400"
+
+	// connection settings.
+	maxIdleConns        = 100
```

Comment:
(very nit) I see we already have the same constants in the `components/event-publisher-proxy/pkg/env/beb_config_test.go`, can we reuse them ?

Reply:
if its in a `_test`-file we cannot. it will only be available in the tests for the other package.

Reply:
Only if we do not export them